data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,142 @@
+
+generic_polllist <- 
+  read_csv("data-raw/congress-generic-ballot/generic_polllist.csv") %>%
+  clean_names() %>%
+  mutate(
+    modeldate = as.Date(modeldate, "%m/%d/%Y"),
+    startdate = as.Date(startdate, "%m/%d/%Y"),
+    enddate = as.Date(enddate, "%m/%d/%Y"),
+    createddate = as.Date(createddate, "%m/%d/%Y"),
+    timestamp = parse_date_time(timestamp, "HMS dmY"),
+    subgroup = as.factor(subgroup),
+    pollster = as.factor(pollster),
+    grade = as.factor(grade),
+    population = as.factor(population),
+    poll_id = as.character(poll_id),
+    question_id = as.character(question_id)
+  )
+devtools::use_data(generic_polllist, overwrite = TRUE)
+
+# generic_topline
+generic_topline <- 
+  read_csv("https://projects.fivethirtyeight.com/generic-ballot-data/generic_topline.csv")
+write_csv(generic_topline, path = "data-raw/congress-generic-ballot/generic_topline.csv")
+
+generic_topline <- 
+  read_csv("data-raw/congress-generic-ballot/generic_topline.csv") %>%
+  clean_names() %>%
+  mutate(
+    modeldate = as.Date(modeldate, "%m/%d/%Y"),
+    timestamp = parse_date_time(timestamp, "HMS dmY"),
+    subgroup = as.factor(subgroup)
+  )
+devtools::use_data(generic_topline, overwrite = TRUE)
+
+
+#inconvenient-sequel ----------------------------------------------------------------
+ratings <- read_csv("data-raw/inconvenient-sequel/ratings.csv") %>%
+  mutate(category = as.factor(category))
+devtools::use_data(ratings, overwrite = TRUE)
+
+
+# mayweather-mcgregor ---------------------------------------------------------------
+tweets <- read_csv("data-raw/mayweather-mcgregor/tweets.csv") %>%
+  mutate(
+    emojis = as.logical(emojis),
+    retweeted = as.logical(retweeted),
+    id = as.character(id)
+  )
+devtools::use_data(tweets, overwrite = TRUE)
+
+
+# mlb-elo ---------------------------------------------------------------------------
+mlb_elo <- read_csv("https://projects.fivethirtyeight.com/mlb-api/mlb_elo.csv")
+write_csv(mlb_elo, path = "data-raw/mlb-elo/mlb_elo.csv")
+
+mlb_elo <- read_csv("data-raw/mlb-elo/mlb_elo.csv") %>%
+  mutate(
+    playoff = as.factor(playoff),
+    neutral = as.logical(neutral)
+  )
+devtools::use_data(mlb_elo, overwrite = TRUE)
+
+
+# soccer-spi ------------------------------------------------------------------------
+#spi_matches
+spi_matches <- read_csv("https://projects.fivethirtyeight.com/soccer-api/club/spi_matches.csv")
+write_csv(spi_matches, path = "data-raw/soccer-spi/spi_matches.csv")
+
+spi_matches <- read_csv("data-raw/soccer-spi/spi_matches.csv") %>%
+  mutate(
+    league_id = as.factor(league_id)
+  )
+devtools::use_data(spi_matches, overwrite = TRUE)
+
+#spi_global_rankings
+spi_global_rankings <- 
+  read_csv("https://projects.fivethirtyeight.com/soccer-api/club/spi_global_rankings.csv")
+write_csv(spi_global_rankings, path = "data-raw/soccer-spi/spi_global_rankings.csv")
+# No changes made
+devtools::use_data(spi_global_rankings, overwrite = TRUE)
+
```

Comment:
Ok, after you make the above changes, the data process is done enough! Don't forget to:
1. Run this entire script so that all the `devtools::use_data(..., overwrite = TRUE)` commands can take the dataframes and save them for use in the R package in `data/DATA_FRAME_NAME.rda`
1. Commit all 10 `data/DATA_FRAME_NAME.rda` files as well, so that they get included in your pull request